src/components/PostDetails/PostDetails.scss

```diff
@@ -7,6 +7,10 @@
     text-align: right;
   }
 
+  &__visiblelist {
```

Comment:
className visibleList shouldn't have "display: none" rule, in such cases we need to create logically connected classNames, such as: 
.visibleList - {display: 'block'}
.hiddenList - {display: 'none'}